tests/test_connector.py

```diff
@@ -0,0 +1,66 @@
+from logging import getLogger
+from unittest import mock
+
+from pytest import fixture, fail
+
+import code42_connector
+from phantom.action_result import ActionResult
+import phantom.app as phantom
+import py42.sdk
+from requests import HTTPError
+
+
+logger = getLogger(name=__name__)
+
+
+NULL_VALUE = object()
+
+
+@fixture
+def py42_sdk(mocker):
+    return mocker.MagicMock(spec=py42.sdk)
+
+
+@fixture
+def py42_client(mocker):
+    return mocker.MagicMock(spec=py42.sdk.SDKClient)
+
+
+@fixture
+def connector(request, py42_client):
+    connector = code42_connector.Code42Connector()
+    connector._client = py42_client
+    return connector
+
+
+def test_handle_connectivity(mocker, connector):
+    param = {}
+    mocker.patch('phantom.base_connector.BaseConnector.add_action_result', return_value=ActionResult(dict(param)))
```

Comment:
Just a style thing, but the other files use double quotes for strings, which I like because they are easier to spot.